### PR TITLE
Cleanup of cluster-config.yaml.tpl

### PR DESCRIPTION
**Pull request by Tomasz-Smelcerz-SAP**
**Description**
Cleanup of **installer-config-cluster.yaml.tpl** file

Changes proposed in this pull request:

- References to **installer-config-cluster.yaml.tpl** removed from documentation.
- Necessary changes to documentation.
- **installer-config-cluster.yaml.tpl** file is removed.

**Related issue(s)**
See also #4329

### CODE REVIEW

File: docs/kyma/05-01-overview.md [hunk 1]
```diff
@@ -3,53 +3,71 @@ title: Overview
 type: Configuration
 ---
```
Comment (piotrmsc) [inline docs/kyma/05-01-overview.md]: @Tomasz-Smelcerz-SAP you've mentioned deletion of cluster config yaml but I cannot see it being deleted here?

Comment (Tomasz-Smelcerz-SAP) [reply docs/kyma/05-01-overview.md]: Removed.

File: docs/kyma/05-01-overview.md [hunk 2]
```diff
@@ -3,53 +3,69 @@ title: Overview
...
+  - Provide overrides that change configuration values used by one or more components
+
+The list of components to install is defined in the **Installation** Custom Resource.
+Overrides are defined as ConfigMap or Secret objects defined by the user before the installation starts.
+The Installer is the Kyma component that is responsible for reading and applying the configuration.
 
-During installation, the Installer applies onto a cluster or Minikube all components defined in the `installer-cr-cluster.yaml.tpl` or `installer-cr.yaml.tpl` templates with their configuration defined in `values.yaml` files. It also imports the configuration overrides defined in the `installer-config-cluster.yaml.tpl` and `installer-config-local.yaml.tpl` templates located under the `installation/resources` subfolder.
 
-> **NOTE:** The installation and configuration templates serve as the basis for creating the installation (`kyma-installer-local.yaml` and `kyma-installer-cluster.yaml`) and configuration (`kyma-config-local.yaml` and `kyma-config-cluster.yaml`) release artifacts.
+## Default settings
```
Comment (jakkab) [inline docs/kyma/05-01-overview.md]: Consider splitting cluster and local sections with a toggle switch.

Comment (Tomasz-Smelcerz-SAP) [reply docs/kyma/05-01-overview.md]: Done.

File: docs/service-catalog/03-04-experimental-features.md [hunk 3]
```diff
@@ -29,18 +29,9 @@ in the installation file:
       service-catalog-apiserver.enabled: "false"
       service-catalog-crds.enabled: "true"
     ```
-- For the cluster installation, add the `service-catalog-overrides` ConfigMap to the [installer-config-cluster.yaml.tpl](https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-config-cluster.yaml.tpl) file: 
+- For the cluster installation, add the `service-catalog-overrides` ConfigMap to the to the cluster before the installation starts. Execute:
```
Comment (mszostok) [inline docs/service-catalog/03-04-experimental-features.md]: duplicated `to the to the`

Comment (Tomasz-Smelcerz-SAP) [reply docs/service-catalog/03-04-experimental-features.md]: fixed.

File: docs/kyma/05-01-overview.md [hunk 4]
```diff
@@ -3,53 +3,80 @@ title: Overview
...
-```
+## Runtime configuration
 
->**NOTE:** You cannot uninstall a Kyma component that is already installed. If you remove it from any of the installation files or add hashtags in front of its `name` and `namespace` entries, you only disable its further updates.
+Changing static configuration of the Kyma after installation is generally not supported.
+Some components may support custom runtime configuration changes though.
+For an example of such a runtime configuration, see [Helm Broker configuration](/components/helm-broker/#configuration-configuration) in which you add links to the bundle repositories to a ConfigMap and label it with the `helm-broker-repo=true` label for the Helm Broker to expose additional Service Classes in the Service Catalog.
 
-Apart from modifying the installation artifacts, you can use ConfigMaps and Secrets to configure the installed components and their behavior in the runtime.
+Another solution for changing components configuration after installation is to alter it's overrides and update the component using Kyma's [Update procedure](#installation-update-kyma).
```
Comment (jakkab) [inline docs/kyma/05-01-overview.md]:
```suggestion
Another solution for changing components configuration after installation is to alter its overrides and update the component using Kyma's [Update procedure](#installation-update-kyma).
```

File: docs/kyma/05-01-overview.md [hunk 5]
```diff
@@ -3,55 +3,81 @@ title: Overview
...
+
+<div tabs>
+  <details>
+  <summary>
+  Local installation
+  </summary>
+
+  The list of available components is stored in Kyma project sources in the file **kyma/installation/resources/installer-cr.yaml.tpl**
+  Default installation overrides can be found in the file **kyma/installation/resources/installer-config-local.yaml.tpl**
+  All the other configuration values are defined directly in components in **/resources** subdirectory of Kyma project.
```
Comment (tomekpapiernik) [inline docs/kyma/05-01-overview.md]:
```suggestion
  Other configuration values are defined directly in components in the configuration of the respective components.
```

File: docs/kyma/05-01-overview.md [hunk 6]
```diff
@@ -3,55 +3,81 @@ title: Overview
...
-Components that are not an integral part of the default Kyma Lite package are preceded with hashtags (#). It means that the Installer skips them during installation.   
-You can customize component installation files by:
-- Removing hashtags in front of the component entries to enable a component installation.
-- Adding hashtags to disable a component installation.
-- Adding new components to the list along with their chart definition in the `kyma/resources` folder. In that case you must create your own [Kyma Installer image](#installation-use-your-own-kyma-installer-image) as you are adding new chart configuration.
+One of the released Kyma artifacts is the Kyma-Installer, a Docker image that combines the Installer executable with charts for all components available in the release.
+The Kyma-Installer can install only the components contained in its image.
+**Installation** Custom Resource specifies which components of all available ones should be actually installed.
+Components that are not an integral part of the default Kyma Lite package are commented out with a hash character (#) and skipped during installation.
+You can customize list of components used for installation by:
```
Comment (tomekpapiernik) [inline docs/kyma/05-01-overview.md]:
```suggestion
You can customize the list of components by:
```

File: docs/kyma/05-01-overview.md [hunk 7]
```diff
@@ -3,55 +3,81 @@ title: Overview
...
+- Uncommenting component entry to enable a component installation.
```
Comment (tomekpapiernik) [inline docs/kyma/05-01-overview.md]:
```suggestion
- Uncommenting a component entry to install the component.
```

File: docs/kyma/05-01-overview.md [hunk 8]
```diff
@@ -3,55 +3,81 @@ title: Overview
...
+- Commenting out an entry using hash chacter to disable a component installation.
```
Comment (tomekpapiernik) [inline docs/kyma/05-01-overview.md]:
```suggestion
- Commenting out a component entry using the hash character (#) to skip the installation of that component.
```

File: docs/kyma/05-01-overview.md [hunk 9]
```diff
@@ -3,55 +3,81 @@ title: Overview
...
+- Adding new components to the list along with their chart definition. In that case you must create your own [Kyma-Installer image](#installation-use-your-own-kyma-installer-image) as you are adding new component to Kyma.
```
Comment (tomekpapiernik) [inline docs/kyma/05-01-overview.md]:
```suggestion
- Adding new components along with their chart definitions to the list. If you do that, you must build your own [Kyma Installer image](#installation-use-your-own-kyma-installer-image) as you are adding a new component to Kyma.
```

File: docs/kyma/05-01-overview.md [hunk 10]
```diff
@@ -3,55 +3,81 @@ title: Overview
...
 
 For more details on custom component installation, see [this](#configuration-custom-component-installation) document.
 
 ### Overrides
 
-The configuration files provide production-ready settings for the Installer. These include ConfigMaps and Secrets with the overrides for the values hardcoded in the `values.yaml` files of component charts.
-
-You can modify local settings, such as memory limits for a given resource, by changing values in the existing ConfigMaps and Secrets or adding new ones. Every ConfigMap and Secret that the Installer reads must contain [obligatory labels](#configuration-helm-overrides-for-kyma-installation-user-defined-overrides).
-
+Common overrides that affect entire installation, for example `global.domainName`, are already described in the [installation procedure](#installation-overview).
```
Comment (tomekpapiernik) [inline docs/kyma/05-01-overview.md]:
```suggestion
The common overrides that affect the entire installation, are described in the installation guides.
```

File: docs/kyma/05-01-overview.md [hunk 11]
```diff
@@ -3,55 +3,81 @@ title: Overview
...
+Other overrides are component-specific.
+To learn about configuration options for a given component, refer to **Configuration** section of the component's documentation.
```
Comment (tomekpapiernik) [inline docs/kyma/05-01-overview.md]:
```suggestion
To learn more about the configuration options available for a specific component, see the **Configuration** section of the component's documentation.
```

File: docs/kyma/05-01-overview.md [hunk 12]
```diff
@@ -3,55 +3,81 @@ title: Overview
...
+Once you know the name and possible set of values for a configuration option, define an override for it by creating a ConfigMap or a Secret. You can also extend an existing one.
```
Comment (tomekpapiernik) [inline docs/kyma/05-01-overview.md]: This sentence is redundant IMO, I'd remove it.

File: docs/kyma/05-01-overview.md [hunk 13]
```diff
@@ -3,55 +3,81 @@ title: Overview
...
 [Read more](#configuration-helm-overrides-for-kyma-installation) about the types of overrides and the rules for creating them.
+>**CAUTION:** An override must exist in a cluster before the installation is started, otherwise the Installer is not be able to apply it correctly.
```
Comment (tomekpapiernik) [inline docs/kyma/05-01-overview.md]:
```suggestion
>**CAUTION:** An override must exist in a cluster before the installation starts. If you fail to deliver create the override before the installation, the configuration can't be applied. 
```

Comment (tomekpapiernik) [reply docs/kyma/05-01-overview.md]: Please add a newline between the "Read more" sentence and the CAUTION panel.

File: docs/kyma/05-01-overview.md [hunk 14]
```diff
@@ -3,55 +3,81 @@ title: Overview
...
 
 > **CAUTION:** In documentation for each Kyma component, you can find configuration documents that list configurable parameters from the `values.yaml` files for a given component chart or sub-chart. Override values only for the parameters exposed in those configuration documents.
 
 ## Runtime configuration
```
Comment (tomekpapiernik) [inline docs/kyma/05-01-overview.md]: Please remove this section

File: docs/service-catalog/03-04-experimental-features.md [hunk 15]
```diff
@@ -29,18 +29,9 @@ in the installation file:
       service-catalog-apiserver.enabled: "false"
       service-catalog-crds.enabled: "true"
     ```
-- For the cluster installation, add the `service-catalog-overrides` ConfigMap to the [installer-config-cluster.yaml.tpl](https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-config-cluster.yaml.tpl) file: 
+- For the cluster installation, add the `service-catalog-overrides` ConfigMap to the cluster before the installation starts. Execute:
```
Comment (tomekpapiernik) [inline docs/service-catalog/03-04-experimental-features.md]:
```suggestion
- For the cluster installation, add the `service-catalog-overrides` ConfigMap to the cluster before the installation starts. Run:
```

File: docs/kyma/05-01-overview.md [hunk 16]
```diff
@@ -3,55 +3,81 @@ title: Overview
 type: Configuration
 ---
 
-## Default settings
+## Introduction
```
[the remark this answers is not in the document]
Comment (Tomasz-Smelcerz-SAP) [reply docs/kyma/05-01-overview.md]: Done.

File: docs/kyma/05-01-overview.md (hunk 12 above)
Comment (Tomasz-Smelcerz-SAP) [reply docs/kyma/05-01-overview.md]: Done.

File: docs/kyma/05-02-custom-component-installation.md [hunk 17]
```diff
@@ -33,7 +33,7 @@ You can add a component before and after installation.
 ### Installation from the release
 
 1. Download the [newest version](https://github.com/kyma-project/kyma/releases) of Kyma.
-2. Customize the installation by adding a component to the list in the installation file or removing hashtags (#) in front of the `name` and `namespace` entries. For example, to enable the Monitoring installation, add or unmark these entries:
+2. Customize the installation by adding a component to the list in the installation file or removing hash charcter (#) in front of the `name` and `namespace` entries. For example, to enable the Monitoring installation, add or unmark these entries:
```
Comment (tomekpapiernik) [inline docs/kyma/05-02-custom-component-installation.md]:
```suggestion
2. Customize the installation by adding a component to the list in the installation file or removing the hash character (#) in front of the `name` and `namespace` entries. For example, to enable the Monitoring installation, add or unmark these entries:
```

File: docs/kyma/05-02-custom-component-installation.md [hunk 18]
```diff
@@ -45,7 +45,7 @@ You can add a component before and after installation.
 
 ### Installation from sources
 
-1. Customize the installation by adding a component to the list of components or removing hashtags (#) in front of the `name` and `namespace` entries in the following installation files:
+1. Customize the installation by adding a component to the list of components or removing hash character (#) in front of the `name` and `namespace` entries in the following installation files:
```
Comment (tomekpapiernik) [inline docs/kyma/05-02-custom-component-installation.md]:
```suggestion
1. Customize the installation by adding a component to the list of components or removing the hash character (#) in front of the `name` and `namespace` entries in the following installation files:
```

File: docs/kyma/05-02-custom-component-installation.md [hunk 19]
```diff
@@ -59,7 +59,7 @@ To add a component that was not installed with Kyma by default, modify the Insta
     ```
     kubectl edit installation kyma-installation
     ```
-2. Add the new component to the list of components or remove hashtags (#) preceding these lines:
+2. Add the new component to the list of components or remove hash character (#) preceding these lines:
```
Comment (tomekpapiernik) [inline docs/kyma/05-02-custom-component-installation.md]:
```suggestion
2. Add the new component to the list of components or remove the hash character (#) preceding these lines:
```

File: docs/kyma/05-02-custom-component-installation.md [hunk 20]
```diff
@@ -76,14 +76,14 @@ You can verify the installation status by calling `./installation/scripts/is-ins
...
 
 ### Installation from the release
 
 1. Download the [newest version](https://github.com/kyma-project/kyma/releases) of Kyma.
-2. Customize the installation by removing a component from the list in the installation file or adding hashtags (#) in front of the `name` and `namespace` entries. For example, to disable the Application Connector installation, remove these entries or add hashtags (#) in front of:
+2. Customize the installation by removing a component from the list in the installation file or adding hash character (#) in front of the `name` and `namespace` entries. For example, to disable the Application Connector installation, remove these entries or add hash character (#) in front of:
```
Comment (tomekpapiernik) [inline docs/kyma/05-02-custom-component-installation.md]:
```suggestion
2. Customize the installation by removing a component from the list in the installation file or adding a hash character (#) in front of the `name` and `namespace` entries. For example, to disable the Application Connector installation, remove these entries or add a hash character (#) in front of:
```

File: docs/kyma/05-02-custom-component-installation.md [hunk 21]
```diff
@@ -95,7 +95,7 @@ You can only remove the component before the installation process starts. To dis
 
 ### Installation from sources
 
-1. Customize the installation by removing a component from the list of components or adding hashtags (#) in front of the `name` and `namespace` entries in the following installation files:
+1. Customize the installation by removing a component from the list of components or adding hash character (#) in front of the `name` and `namespace` entries in the following installation files:
```
Comment (tomekpapiernik) [inline docs/kyma/05-02-custom-component-installation.md]:
```suggestion
1. Customize the installation by removing a component from the list of components or adding a hash character (#) in front of the `name` and `namespace` entries in the following installation files:
```

File: docs/kyma/05-01-overview.md (hunk 14 above)
Comment (Tomasz-Smelcerz-SAP) [reply docs/kyma/05-01-overview.md]: Done.

**Pull request merged**